thundra/integrations/mongodb.py

```diff
@@ -0,0 +1,112 @@
+import traceback
+import json
+import time
+import logging
+from thundra import config, constants
+from thundra.plugins.invocation import invocation_support
+from thundra.integrations.base_integration import BaseIntegration
+from thundra.opentracing.tracer import ThundraTracer
+
+try:
+    from pymongo.monitoring import CommandListener
+    from bson.json_util import dumps
+except ImportError:
+    pass
+
+
+logger = logging.getLogger(__name__)
+
+
+class CommandTracer(CommandListener):
```

Comment:
There is also a possibility that `CommandListener` does not exist if the above try-except block ended in except part.